crates/gateway-client/src/lib.rs

```diff
@@ -22,7 +22,7 @@ mod metrics;
 #[allow(unused_variables)]
 #[cfg_attr(feature = "test-utils", mockall::automock)]
 #[async_trait::async_trait]
-pub trait GatewayApi: Sync {
+pub trait GatewayApi: Send + Sync {
```

Comment:
:eyes: This probably shouldn't be here and is some side effect of me trying to get every commit to build while rebasing the ws PR.

Reply:
Yep, commit removed.